packages/eslint-config-react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@side/eslint-config-react",
-  "version": "0.8.5",
+  "version": "0.9.0",
```

Comment:
You don't manage the version. The CI will take care of this automagically based on the commit message that is merged into master.